config/grafana/dashboards/integration-service-dashboard.json

```diff
@@ -616,7 +616,7 @@
         {
           "editorMode": "code",
           "exemplar": true,
-          "expr": "sum(rate(release_latency_seconds_bucket{le=10}[$__rate_interval])) by (job)/sum(rate(release_latency_seconds_bucket[$__rate_interval])) by (job)*100",
```

Comment:
Why is this change in this PR?

Reply:
I had a merge conflict from a PR I merged earlier today, despite doing a pull. I have it fixed up now. I'll keep an eye to  make sure it all goes through ok.